tests/connect/replicaset-no-replicaset-name-001.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+MongoDB\Driver\Manager: Connecting to Replica Set with an empty replicaSet argument
```

Comment:
Can we rename this file to `manager-ctor_error-003.phpt`? I realize we have a lot of existing tests split across various sub-directories, but the general convention is `class-method-###.phpt`, with an optional `_error` for error tests.